Sources/XCTest/XCTestCase.swift

```diff
@@ -189,7 +189,7 @@ extension XCTestCase {
     ///   between these environments. To ensure compatibility of tests between
     ///   swift-corelibs-xctest and Apple XCTest, it is not recommended to pass
     ///   explicit values for `file` and `line`.
-    public func expectationWithDescription(description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
+    public func expectation(withDescription description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
```

Comment:
It was a discussion we had in our Swift 3 naming 'working group'.

The summary is that we felt that the importer could not automatically make the right decision about if a 'with' was important or not in an API. It is used to mean different things in different APIs. Therefore we decided to stick with a default of leaving it in place for imported API.

We can of course change our minds about the names, but then we're talking about changing it for all clients on all platforms. Since this version of XCTest is attempting to keep the same API, and there isn't a plan to change the Darwin version of this method, we should keep the name the same for now.